src/theme/theme.ts

```diff
@@ -357,6 +357,8 @@ export const darkTheme: Theme = {
     primaryMain: palette.black,
     secondaryMain: palette.blueGray,
   },
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
I think we should just remove the dark theme entirely. Doesn't seem to be something we're going to prioritize.

Reply:
Sounds good, didnt seem like it was used anywhere?